tests/platform_tests/test_cpu_memory_usage.py

```diff
@@ -36,7 +36,8 @@ def setup_thresholds(duthosts, enum_rand_one_per_hwsku_hostname):
     high_cpu_consume_procs = {}
     is_asan = is_asan_image(duthosts, enum_rand_one_per_hwsku_hostname)
     if duthost.facts['platform'] in ('x86_64-arista_7050_qx32', 'x86_64-kvm_x86_64-r0', 'x86_64-arista_7050_qx32s',
-                                     'x86_64-cel_e1031-r0', 'x86_64-arista_7800r3a_36dm2_lc') or is_asan:
+       'x86_64-cel_e1031-r0') or is_asan or \
+       ('arista_7800' in duthost.facts['platform'].lower()):
```

Comment:
why do we need to increase to 90% if the steady state memory util is 60%? Can we increase it to 75%?

Reply:
The initial thought was to set it at 90 as a preventative measure to ensure we don't hit the limit. But we can set it at 75%